src/pages/blog/articles/_all-articles.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import ArticleCard from './_article-card'
+import FeaturedArticle from './_featured-article'
+import { Container, Flex } from 'components/containers'
+import { Text, LocalizedLinkText } from 'components/elements'
+import { localize } from 'components/localization'
+import RightArrow from 'images/svg/black-right-arrow.svg'
+
+const VideoGrid = styled.div`
+    display: grid;
+    width: 100%;
+    height: 100%;
+    margin: 40px 0;
+    grid-template-columns: repeat(auto-fit, minmax(288px, 384px));
+    grid-row-gap: 40px;
+    grid-column-gap: 24px;
+    grid-template-rows: auto;
+`
+
+const AllArticles = ({ article_data }) => {
+    return (
+        <Container fd="column">
+            <Flex jc="flex-start" ai="center" mt="4rem">
+                <LocalizedLinkText to="/blog" color="grey-5">
+                    {localize('Home')}
```

Comment:
Are we planning to localize blog?

Reply:
Probably not in Phase 1 but using it shouldn't be an issue if I'm not wrong right?

Reply:
I'm afraid it would add more strings for translation process later and get confusing for translators/us? Also we might need to set redirects for all other langs to en

Reply:
I'll remove it for now then 👍